doc/release/1.11.0-notes.rst

```diff
@@ -39,6 +39,13 @@ DeprecationWarning to error
 FutureWarning to changed behavior 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* Multidimensional indexing with anything but a base class tuple is
+  deprecated. This means that code such as ``arr[[slice(None)]]`` has to
+  be changed to ``arr[tuple([slice(None)])]``. This is necessary to avoid
```

Comment:
Hehe, yeah, though that would somewhat defeat the purpose ;). should make the example longer... Anyway, doesn't look like 1.11 change.
Hmmm, reading my last comment, part of me wonders why the masked arrays are always so tricky when doing indexing changed.